sql/core.go

```diff
@@ -713,7 +713,7 @@ type StatusVariableRegistry interface {
 	// SetGlobal sets the global value of the status variable with the given name, returns an error if the variable is SessionOnly scope
 	SetGlobal(name string, val interface{}) error
 	// IncrementGlobal increments the value of the status variable by the given integer value
```

Comment:
Should document failure behavior, kind of important

Reply:
added note about session-only nooping